Prepare repo for bb tests (#42)
CoherentLabs/Gameface-UI: docs/src/content/docs/concepts/tests.mdx

## Running the tests

To run the tests, you'll first need to have Gameface's `Player.exe` available on your machine. Once that's in place, there are two ways to run the tests from the command line:

### Inline the `Player.exe` path in the command

You can run the tests by directly passing the path to `Player.exe` as an environment variable in the command line. From the root directory of the Gameface UI project, run:

```bash
npx cross-env GAMEFACE_PATH={absolute_or_relative_path_to_Player.exe} npm run test
```

This command sets the `GAMEFACE_PATH` environment variable, which tells the test runner where to find the `Player.exe`. While both absolute and relative paths are accepted, using an absolute path is recommended to prevent issues related to incorrect path resolution. If you use a relative path, it should be relative to the root of the Gameface UI project.

### Using `.env` file :badge[recommended]{variant="note"}

If you run tests frequently, configuring the path via a `.env` file can be more convenient. Create a `.env` file in the root of the Gameface UI project with the following content:

```bash title=".env"

> GAMEFACE_PATH={absolute_or_relative_path_to_Player.exe}

Author: Added a small example  here

```

Once this file is in place, you can simply run:

```bash
npm run tests
```

:::caution

> Make sure you have ran `npm i` command before running the tests to install all the required dependencies.

Reviewer: ```suggestion
Make sure you have run `npm i` in the root of the repository before running the tests to install all required dependencies.
```
